xml/deploy_cephadm.xml

```diff
@@ -643,7 +643,7 @@ o- time_server ................................................ [enabled]
      If there is no default path set or your deployment requires a specific
      path, add it as follows:
     </para>
-<screen>&prompt.smaster;ceph-salt config /containers/images/ceph set registry.suse.com/ses/7/ceph/ceph</screen>
+<screen>&prompt.smaster;ceph orch upgrade ceph/ceph:latest</screen>
```

Comment:
1. `registry.suse.com/ses/7/ceph/ceph:latest`  is already correct
2. `ceph-salt config /containers/images/ceph set registry.suse.com/ses/7/ceph/ceph` is actually correct as well. 

`ceph orch upgrade --image registry.suse.com/ses/7/ceph/ceph:latest` initiates an update to the next minor version. that should go into the admin guide.